scripts/upgrade

```diff
@@ -90,16 +97,14 @@ ynh_install_app_dependencies $pkg_dependencies
 # CREATE DIRECTORIES
 #=================================================
 
-mkdir -p /home/yunohost.transmission/{progress,completed,watched}
+datadir=/home/yunohost.app/$app
```

Comment:
```suggestion
```

That should not be needed considering the var is either fetched from the setting or auto-defined in the migration snippet earlier